pom.xml

```diff
@@ -4964,6 +4964,12 @@
         <version>0.2.5</version>
      </dependency>
 
+     <dependency>
+        <groupId>org.awaitility</groupId>
+        <artifactId>awaitility</artifactId>
+        <version>3.1.6</version>
```

Comment:
Should we scope it `test` directly here?

Reply:
I'd say it's better to keep only the version management in depencenyManagement and explicitely set the test scope in the project poms.